etc/DependencyInstaller.sh

```diff
@@ -12,9 +12,8 @@ _installCommonDev() {
     swigVersionType="tag"
     swigVersion=4.1.0
     swigChecksum="794433378154eb61270a3ac127d9c5f3"
-    boostVersionBig=1.80
-    boostVersionSmall=${boostVersionBig}.0
-    boostChecksum="077f074743ea7b0cb49c6ed43953ae95"
+    boostVersionBig=${boostVersion%.*}
+    boostVersionSmall=${boostVersion}
```

Comment:
Let's drop `boostVersionSmall` and replace with `boostVersion`. And move `boostVersionBig` to line ~58.